### DISCUSSION TITLE
Odd day'd weeks.

### DISCUSSION THREAD

**Discussion by LoopRook** [vigoren/foundryvtt-simple-calendar#150] [Ideas]
It would be great to be able to have some weeks have an extra day. While odd, some game settings use this like The Scarred Lands. Every third week has an 8th day which is unique to that week. 

Aka , Mon Tues Weds Thurs Fri Sat Sun for 2 weeks, and on the third after Sun is a unique day.

**Comment by vigoren** [comment]
That's a neat idea. I can look into adding something like this for sure 😁

**Comment by grungydan** [comment]
Another vote from me. Coriolis: The Third Horizon uses a calendar of nine "months" with three "holidays" between segments of three months. 

https://i.imgur.com/UN1vZp7.png

In addition to that, the "months" are all 37 days long. A very popular [community project](https://preview.drivethrurpg.com/en/product/324228/The-Coriolis-Calendar) suggests breaking the months into "novenas" of 9 days each, and tacking a "not day" on at the end of the month to account for the 37th day. 

Here's how I've kludged it into Simple Calendar for now: 

https://github.com/grungydan/coriolis-simple-calendar/blob/main/Novena-based-Coriolis-Calendar-simple-calendar.json

In addition to allowing us to have one week that's a day longer (as would be possible by OP's request), it would be amazing if we could set Week Names the way we can set Year Names. In the above linked Calendar from DriveThru, it is suggested that people say "<name of the week> 1, <name of the week> 2...." This would allow the creation or importation of calendars in which days of the week aren't named in the Western fashion. (Example: the names of the days of the week in Mandarin are literally translated as "day of the week one, day of the week two...."

Example date for the setting: Today is two water Deckhand, CC61 
Translation: it is the second day of the second week (which is the week named the Novena of Water), Coriolis Cycle 61.

**Reply by grungydan** [reply]
Rereading that it might be simple for me to just ask for the ability to name weeks! :)

**Reply by vigoren** [reply]
This is something I can look into for sure. I do have another task for named weeks that I will be adding in the future.

Right now I am finishing up a pretty large update (took the holidays off from working on it) then will be jumping into a bunch of week and day specific updates!
